components/eamxx/scripts/test_factory.py

```diff
@@ -176,7 +176,7 @@ def __init__(self, _):
             "debug with compute sanitizer memcheck",
             [("CMAKE_BUILD_TYPE", "Debug"),
              ("EKAT_ENABLE_COMPUTE_SANITIZER", "True"),
-             ("EKAT_COMPUTE_SANITIZER_OPTIONS", "--tool=memcheck")],
+             ("EKAT_COMPUTE_SANITIZER_OPTIONS", "'--tool=memcheck'")],
```

Comment:
These changes are unrelated, but if we ever need to add more options to these tests, not having these quotes might lead to hard to find errors. But I can revert this commit if anyone wants.

Reply:
See my comment above. I think something is not right.